src/lib/components/forms/form/form-components/slider/slider.component.html

```diff
@@ -22,3 +22,9 @@
 <mat-error *ngIf="isRequired && control.hasError('required')">
     {{ 'validators.required' | translate: _locale.language }}
 </mat-error>
+<mat-error *ngIf="min && control.hasError('min')">
```

Comment:
Gefährlich.. min kann halt auch 0 sein.. müsste stattdessen also überprüft werden ob es nicht null oder undefined ist.
Oder du lässt das komplett aus der Bedingung raus. Der zweite Teil trifft ja eh nur zu wenn der validator auch gesetzt ist.

```suggestion
<mat-error *ngIf="control.hasError('min')">
```